cg/services/deliver_files/delivery_file_fetcher_service/error_handling.py

```diff
@@ -14,10 +15,20 @@ def handle_missing_bundle_errors(func):
 
     @wraps(func)
     def wrapper(*args, **kwargs):
+        case_id: str = kwargs.get("case_id")
+        sample_id: str = kwargs.get("sample_id")
+        case: Case = kwargs.get("case")
         try:
             return func(*args, **kwargs)
-        except HousekeeperBundleVersionMissingError as error:
-            LOG.error(error)
+        except HousekeeperBundleVersionMissingError:
+            msg: str = f"Missing bundles detected for:"
+            if case_id:
+                msg += f" case {case_id}"
+            if sample_id:
+                msg += f" sample {sample_id}"
+            if case:
```

Comment:
A bit confused by the possibility of both a case_id and a case here. But if not important, feel free to resolve.

Reply:
two decorators that get called in the appropraite place